modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/utils/setFileFormik.ts

```diff
@@ -0,0 +1,66 @@
+/* The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+const getFileReader = (file: any) => {
```

Comment:
Try to use FileList interface

```suggestion
const getFileReader = (file: FileList) => {
```